Modules/Server/Sources/PocketCastsServer/Public/Refresh/RefreshOperation.swift

```diff
@@ -133,27 +128,37 @@ class RefreshOperation: Operation {
                 newEpisode.episodeStatus = DownloadStatus.notDownloaded.rawValue
                 newEpisode.addedDate = Date()
                 newEpisode.populate(fromEpisode: episode)
-                DataManager.sharedManager.save(episode: newEpisode)
+                return newEpisode
+            })
 
-                newEpisodesAdded += 1
+            DataManager.sharedManager.bulkSave(episodes: episodes)
```

Comment:
Instead of saving the episode each time, this bulk saves all episodes using a single query.